nix/overlay.nix

```diff
@@ -384,24 +384,14 @@ in {
       pkg = final.unchecked.tenzir-de.override {
         pname = "tenzir";
       };
-    in
-      pkg.withPlugins (ps: [
-        ps.context
-        ps.matcher
-        ps.pipeline-manager
-        ps.platform
-      ]);
-    tenzir-ee = let
-      pkg = final.unchecked.tenzir-de.override {
-        pname = "tenzir-ee";
-      };
     in
       pkg.withPlugins (ps: [
         ps.compaction
         ps.context
         ps.matcher
         ps.pipeline-manager
         ps.platform
+        ps.vast
```

Comment:
Why do we need to define all the plugins here again when they're already defined here? https://github.com/tenzir/tenzir/blob/topic/unify-builds/nix/tenzir/plugins/names.nix

Reply:
Now that we have the complete list here we can do that, but we couldn't before when we had individual edition builds. This code could be refactored to reflect that change, but I didn't have the time to do it yet.

Reply:
Okay, I'll write a story for this.